src/main/webapp/WEB-INF/views/manufacturers/add.jsp

```diff
@@ -10,6 +10,7 @@
 <body>
 <form method="post" id="manufacturer" action="${pageContext.request.contextPath}/manufacturers/add"></form>
 <h1 class="table_dark">Add manufacturer:</h1>
+<%@include file="/WEB-INF/views/header.jsp"%>
```

Comment:
Header should be on the top of the site. Set it above all other mark up elements, wherever possible.